cudapoa/src/batch.cu

```diff
@@ -52,33 +54,36 @@ std::unique_ptr<Batch> create_batch(int32_t device_id,
                                                                     (int32_t)gap_score,
                                                                     (int32_t)mismatch_score,
                                                                     (int32_t)match_score,
-                                                                    cuda_banded_alignment);
+                                                                    cuda_banded_alignment,
+                                                                    cuda_adaptive_alignment);
         }
         else
         {
-            return std::make_unique<CudapoaBatch<int32_t, int16_t>>(device_id,
+            return std::make_unique<CudapoaBatch<int32_t, int32_t>>(device_id,
                                                                     stream,
                                                                     max_mem,
                                                                     output_mask,
                                                                     batch_size,
                                                                     (int32_t)gap_score,
                                                                     (int32_t)mismatch_score,
                                                                     (int32_t)match_score,
-                                                                    cuda_banded_alignment);
+                                                                    cuda_banded_alignment,
+                                                                    cuda_adaptive_alignment);
         }
     }
     else
     {
         // if ScoreT is 16-bit, then it's safe to assume SizeT is 16-bit
-        return std::make_unique<CudapoaBatch<int16_t, int16_t>>(device_id,
+        return std::make_unique<CudapoaBatch<int32_t, int32_t>>(device_id,
```

Comment:
I think this carried over from my debugging changes.

Reply:
good catch. fixed.